bitcomplete/complete.go

```diff
@@ -2,15 +2,27 @@
 package main
 
 import (
+	"fmt"
 	"github.com/c-bata/go-prompt"
 	"github.com/chriswalz/bit/cmd"
 	"github.com/posener/complete/v2"
 	"github.com/posener/complete/v2/predict"
 	"github.com/thoas/go-funk"
+	"os"
+	"strconv"
 	"strings"
 )
 
 func main() {
+	comp_line := os.Getenv("COMP_LINE")
+	comp_point := os.Getenv("COMP_POINT")
+	i, err := strconv.Atoi(comp_point)
+	if err != nil {
+		fmt.Println("COMP_LINE", comp_line, "COMP_POINT", comp_point, "err:", err)
```

Comment:
I would recommend returning here and adding a  bit more  information on the log for the user. As it's unlikely that the user would be aware of the environment variables